cmake@3.13.rb

```diff
@@ -13,8 +13,6 @@ class CmakeAT313 < Formula
   # CMake is built with Qt support and Qt is built with MySQL support as MySQL uses CMake.
   # For the GUI application please instead use `brew cask install cmake`.
 
-  needs :cxx11
```

Comment:
Why is it not needed?

Reply:
Newer versions of brew dont need it any longer as they have removed support for older compilers. As references see:
https://github.com/Homebrew/brew/pull/5599
https://github.com/Homebrew/homebrew-core/pull/36362